build/AzureInfo.props

```diff
@@ -1,6 +1,10 @@
 <Project>
   <PropertyGroup>
     <Product>Sdk</Product>
+    <CoreSetupBlobRootUrl>$(PB_AssetRootUrl)</CoreSetupBlobRootUrl>
+    <CoreSetupBlobRootUrl Condition="'$(CoreSetupBlobRootUrl)' == ''">https://dotnetcli.azureedge.net/dotnet/</CoreSetupBlobRootUrl>
+    <CoreSetupBlobAccessToken>$(PB_AssetRootAccessTokenSuffix)</CoreSetupBlobAccessToken>
```

Comment:
Should this be condition on CoreSetupBlobAccessToken being empty?

Reply:
Fixed...